### DISCUSSION TITLE
Use Standard features only?

### DISCUSSION THREAD

**Discussion by Claude-at-Instaclustr** [rrevenantt/antlr4rust#43] [Q&A]
@rrevenantt 

Is there a plan to migrate your code to a standard feature only implementation (e.g. no CoerceUnsized)?  I our environment we can only use the release of Rust.  Any assistance would  be appreciated.

Claude

**Comment by rrevenantt** [comment]
Yes, on next version which is WIP in #29 and already works on stable rust. I am going publish a prerelease version soon.
